frame/support/src/remove_pallet.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017-2023 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright Parity Technologies (UK) Ltd.
```

And could this please be made a module of `migrations`?